cvat/apps/dataset_manager/bindings.py

```diff
@@ -1885,17 +1885,21 @@ def load_dataset_data(project_annotation, dataset: dm.Dataset, project_data):
             'data_root': dataset.data_path + osp.sep,
         }
 
+        data_root = None
         for dataset_item in subset_dataset:
             if dataset_item.image and dataset_item.image.has_data:
                 dataset_files['media'].append(dataset_item.image.path)
+                if data_root is None:
+                    data_root = osp.splitext(dataset_item.image.path)[0].replace(dataset_item.id, '')
```

Comment:
This could corrupt the path if its leading part happens to also contain the ID. You must make sure you're only removing the ID from the end of the path.

Reply:
I'd also suggest to use just `osp.split` instead.